modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/GraphQLServletTest.java

```diff
@@ -458,48 +453,25 @@ public void testSchema() throws Exception {
 		_assertGraphQLSchemaField(
 			false, mutationFieldsJSONArray, true, "testPath_v1_0");
 		_assertGraphQLSchemaField(
-			false,
-			JSONUtil.getValueAsJSONArray(
-				_getJSONObject(mutationFieldsJSONArray, "testPath_v1_0"),
-				"JSONObject/type", "JSONArray/fields"),
-			true, "createTestDTO");
+			true, mutationFieldsJSONArray, true, "createTestDTO");
 
 		// Query fields
 
-		JSONArray queryFieldsJSONArray = JSONUtil.getValueAsJSONArray(
+		JSONArray typeFieldsJSONArray = JSONUtil.getValueAsJSONArray(
```

Comment:
You removed the assertion over the `queryType` graphQL field. The type has to be done for both `mutationType` and `queryType`